tests/unit/test_utils.py

```diff
@@ -0,0 +1,312 @@
+from pathlib import Path, PosixPath
+from shutil import rmtree
+from typing import Generator, Tuple
+from zipfile import ZipFile
+
+import pytest
+import requests_mock
+import typer
+from snowcli import utils
+
+from tests.unit.test_data.test_data import *
+
+# TODO: check for consistency in using ' or "
+
+
+class TestUtils:
+
+    APP_ZIP = "app.zip"
+    CORRECT_METADATA = "correct_metadata.yaml"
+    FILE_IN_A_SUBDIR = "file_in_a_subdir.txt"
+    FILE_IN_SECOND_TEST_DIRECTORY = "very_important_file.txt"
+    REQUIREMENTS_SNOWFLAKE = "requirements.snowflake.txt"
+    REQUIREMENTS_TXT = "requirements.txt"
+    SECOND_TEST_DIRECTORY = "other_test_dir"
+    SUBDIR = "subdir"
+    TEMP_DIR_FOR_APP_ZIP = "temp_dir"
+    TEMP_TEST_DIRECTORY = ".tests"
+
+    @pytest.mark.parametrize("argument", utils.YesNoAskOptions)
+    def test_yes_no_ask_callback_with_correct_argument(self, argument: str):
+        result = utils.yes_no_ask_callback(argument)
+
+        assert result == argument
+
+    @pytest.mark.parametrize("argument", bad_arguments_for_yesnoask)
```

Comment:
Do we use `bad_arguments_for_yesnoask` more than once? In tests my personal preference is to keep parametrize input inline so developers can easily see what's being tests. WDYT?

Reply:
Idea behind this, was to separate all the test data from test, to make everything clearer. Probably, in this case it would be easier to understand if the list was passed directly, but on the other hand it would mean mixing both solutions - if we`re ok with that, i`ll pass the argument directly

Reply:
Mixing is fine with me